### ISSUE TITLE
Is there a feed for new releases? 

### ISSUE THREAD

**Issue by zblesk** [huginn/huginn#2374]
I use Huginn to compile a mail with info about new releases for self-hosted SW I use. I'd also like to be notified about updates to Huginn itself, but there are 0 releases on Github, and Docker Hub doesn't offer any feed functionality. 

Is there a place where I can get info about new releases? If not, is there a plan to provide one? 

Thanks.

**Comment by dsander** [comment]
We don't do releases, but we also try our hardest to avoid breaking changes, so in a way every commit to master is a new release 😄 The docker image is rebuild on CI for every commit to master.

GitHub offers an RSS for commits: https://github.com/huginn/huginn/commits/master.atom

**Comment by zblesk** [comment]
Ah, I see. You guys are hard core. :D  

Thanks, I'll look into that. Now just to figure out how to squash a stream of RSS updates into a single "yes, there has been an update". (I'll look at the Digest.) 
Speaking of which: is there some forum, or other place, where I can just ask about things that aren't _issues_ per se?

**Comment by dsander** [comment]
Yep the DigestAgent is perfect if you only want to get updates every day or week.



> Speaking of which: is there some forum, or other place, where I can just ask about things that aren't _issues_ per se?

The GitHub issues are for everything, feature request, real bugs and questions. We also have a [Gitter](https://gitter.im/huginn/huginn) chat (which I am not following actively because I like the "forum" style of issues more to answer questions).
